### ISSUE TITLE
Exports and `object-curly-spacing`

### ISSUE THREAD

**Issue by valeriangalliat** [babel/babel-eslint#167]
Hi,

With the following code:

```js
export x from 'x';
```

And `"object-curly-spacing": [2, "never"]`, I get the following error:

```
There should be no space after 'export' (object-curly-spacing)
There should be no space before 'from' (object-curly-spacing)
```

I guess it's because [ESLint doesn't support the export syntax without braces](https://github.com/eslint/eslint/blob/649366b95bbb502a065f1d399ee2488b9ff3a67f/lib/rules/object-curly-spacing.js#L192), but I'm not sure where to fix this. I did fix it [on ESLint side](https://github.com/valeriangalliat/eslint/commit/5657b9cf618bd1ee1270fd827e9718fcb6e54cfd) but I'm not sure what to do with this, since this syntax is not supported by the default ESLint parser...

Any idea? Do you think I should open an issue and a PR on ESLint with this fix?
<sup><sub>I'm interested in @nzakas opinion on this too.</sub></sup>

**Comment by sebmck** [comment]
Can you confirm whether or not you get this error on default ESLint please.

**Comment by hzoo** [comment]
Looks like `Illegal export declaration` on default - so if the syntax should be supported as a part of es2015 then you should report it to espree and make a PR to ESLint, otherwise you should use your own plugin rule / send a PR to https://github.com/babel/eslint-plugin-babel?

**Comment by hzoo** [comment]
Actually looking at it now - eslint-plugin-babel already has that change - closing in favor of using that.

> babel/object-curly-spacing: doesn't complain about export x from "mod"; or export * as x from "mod";

**Comment by valeriangalliat** [comment]
Oh, I didn't know about eslint-plugin-babel, awesome, thank you!
